doc/man3/OSSL_HPKE_CTX_new.pod

```diff
@@ -84,48 +84,159 @@ is likely required before using these APIs.  HPKE is used by various
 other IETF specifications, including the (draft) TLS Encrypted Client 
 Hello (ECH) specification and others.
 
-HPKE is a highly flexible consstruct requiring supporting many optional
-inputs for the various combinations in which modes and algorithms can
-be used.
-
-Notes on the various operations supported:
-
-- OSSL_HPKE_seal() is used by senders to encrypt a value to a
-recipient (supplying optional additional authenticated data or
-AAD)
+HPKE is a highly flexible consstruct supporting many optional inputs for the
+various combinations in which modes and algorithms can be used.
+
+=head2 Define'd symbols and data strutures
+
+- Symbols are defined (in include/openssl/hpke.h) for each of the
```

Comment:
Probably dont need to specify where they come from..

Reply:
well, there isn't a listing of 'em in the pod file (I guess it'd be too long to be useful), so seemed better to point at where the list is, but no problem dropping that